spec/warden/jwt_auth/env_helper_spec.rb

```diff
@@ -43,6 +53,18 @@
 
       expect(updated_env['HTTP_AUTHORIZATION']).to eq('Bearer 123')
     end
+
+    describe 'custom token header' do
+      it 'sets value as configured token_header' do
+        allow(Warden::JWTAuth.config).to receive(:token_header).and_return('Test_Authorization')
+
+        env = {}
+
+        updated_env = described_class.set_authorization_header(env, 'Bearer 123')
+
+        expect(updated_env['HTTP_TEST_AUTHORIZATION']).to eq('Bearer 123')
+      end
+    end
```

Comment:
I felt it was worth adding specs specifically testing default/custom config behaviour due to the potential for this to cause issues to existing projects using this gem.

If preferred, it can be updated to work in the same way as the aud_header test below

Reply:
No worries, I think there's no need. Let's be consistent with how we're testing `aud_header`.

Reply:
Thanks!